Tasks/InstallAppleCertificateV2/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 149,
+        "Minor": 150,
```

Comment:
Since you're touching iOS common, you'll also have to bump InstallAppleProvisioningProfile and Xcode

Reply:
The PR build caught that error which is good!